features/admin/product/managing_product_options/adding_product_option.feature

```diff
@@ -41,3 +41,20 @@ Feature: Adding a new product option
         Then I should be notified that it has been successfully created
         And the product option "T-Shirt size" should appear in the registry
         And product option "T-Shirt size" should have the "S" option value
+
+    @no-api @ui @javascript
+    Scenario: Adding a new product option with applying option value to all option values
+        Given the store is also available in "Polish (Poland)"
+        When I want to create a new product option
+        And I name it "T-Shirt size" in "English (United States)"
+        And I specify its code as "t_shirt_size"
+        And I add the "S" option value identified by "OV1" in "English (United States)"
+        And I apply the option value identified by 'OV1' in 'English (United States)' to all option values.
+        And I add the "M" option value identified by "OV2" in "English (United States)"
+        And I add it
+        Then I should be notified that it has been successfully created
+        And the product option "T-Shirt size" should appear in the registry
+        And product option "T-Shirt size" should have the "S" option value in "English (United States)" locale
+        And product option "T-Shirt size" should have the "M" option value in "English (United States)" locale
+        And product option "T-Shirt size" should have the "S" option value in "Polish (Poland)" locale
+        But product option "T-Shirt size" should not have the "M" option value in "Polish (Poland)" locale
```

Comment:
You could save the product option in shared storage in the `... in registry` step, and then just `this product option should...`, it would be a bit more readable

Reply:
In the `... in the registry` step, there is only the name of product option, but according to you concept, I need an entity. Therefore, I think it is unnecessary

Reply:
The same applies to the `product option "T-Shirt size" should have the "S" option value` steps, the only difference is you're hitting the repo in each step instead of getting it once and saving it in the shared storage